runtime/centrifuge/src/lib.rs

```diff
@@ -1065,6 +1614,19 @@ construct_runtime!(
 		OrmlTokens: orml_tokens::{Pallet, Storage, Event<T>, Config<T>} = 151,
 		OrmlAssetRegistry: orml_asset_registry::{Pallet, Storage, Call, Event<T>, Config<T>} = 152,
 		OrmlXcm: orml_xcm::{Pallet, Storage, Call, Event<T>} = 153,
+
+		// Synced pallets across all runtimes - Range: 180-240
+		// WHY: * integrations like fireblocks will need to know the index in the enum
```

Comment:
Not a blocker to merging this, but I wonder if it would be better for us to expose EVM precompiles for usecases that can't parse metadata. We'd then  have:

* Stable(-ish) Substrate APIs through native pallet calls, where we can do things like reorder enums, add error variants, or insert new calls
* 100% stable precompiles, where we'd introduce a new precompile version for any change that would require re-parsing metadata.

I'm very worried about the implications for our ability to develop our pallets if we need to maintain compatibility for users who can't re-parse metadata quickly after an update.

Reply:
XCM can invoke a precompile through the EVM pallet relatively easily. I haven't fully though through the tradeoffs. I'm just very familiar with the pain that maintaining ABI compatibility can be vs just maintaining API compatibility. Precompiles have a much larger namespace (they're just contract addresses), so we have an opportunity to let certain ABI levels "freeze" in a way we can't do with pallets.

Reply:
SOunds like it is not a bad idea, but I think it will be really hard to have the correct `AccountId` mappings then. As we always derive them. Lets keep that in mind.